libtiledbsc/include/tiledbsc/column_buffer.h

```diff
@@ -3,9 +3,15 @@
 
 #include <stdexcept>  // for windows: error C2039: 'runtime_error': is not a member of 'std'
 
+// TODO: pybind11 used for intermediate testing, remove after adding arrow
```

Comment:
Do we want pybind11 in these functions? In TileDB-VCF pybind11 is only used in the python API files?

Reply:
Right, we don't want pybind11 here. It was useful for initial experiments/tests and will be removed with the PR that adds arrow support.